### DISCUSSION TITLE
Ignore some properties

### DISCUSSION THREAD

**Discussion by etrange02** [Gremlinq/ExRam.Gremlinq#599] [Ideas]
Hi!

First of all, this library is very nice to use! It is so easy to request Janusgraph with it instead of other libs.

I am facing a problem with my models : some properties must not be added into GraphSON data. I set `schema.default = none`.

So, I wonder if a GraphsonIgnoreAttribute can be an interesting addition.

Thank you,
David L.

**Comment by danielcweber** [comment]
[This unit test shows the way.](https://github.com/ExRam/ExRam.Gremlinq/blob/8ec40d7505568232a3df02f6d1d5edee3a492fd8/test/ExRam.Gremlinq.Core.Tests/QueryExecutionTest.cs#L301)
